incubator/haproxy-ingress/values.yaml

```diff
@@ -31,6 +31,13 @@ controller:
     tag: "v0.5-beta.3"
     pullPolicy: IfNotPresent
 
+  defaultSslCertificate:
```

Comment:
This section should be added in READNE.md file .

Reply:
done. I also have squashed the commits.